hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/TransactionalMapCodecTemplate.java

```diff
@@ -257,4 +257,16 @@
      */
     @Request(id = 17, retryable = false, response = ResponseMessageConst.LIST_DATA)
     Object valuesWithPredicate(String name, String txnId, long threadId, Data predicate);
-}
+
+    /**
+     * Returns true if this map contains an entry with the value.
+     *
+     * @param name     Name of the Transactional Map
+     * @param txnId    ID of the this transaction operation
+     * @param threadId The id of the user thread performing the operation. It is used to guarantee that only the lock holder thread (if a lock exists on the entry) can perform the requested operation.
+     * @param value    The value whose existence is checked.
+     * @return True if this map contains an entry for the specified key.
+     */
+    @Request(id = 18, retryable = false, response = ResponseMessageConst.BOOLEAN)
```

Comment:
Missing Since tag

Reply:
Hi @MatCyg 
Can you add a since tag with ` @Since(value = "1.6")`  here ?

For a sample usage you can look at here,
https://github.com/hazelcast/hazelcast-client-protocol/blob/master/hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ClientMessageTemplate.java#L370